ketcher-autotests/tests/Settings/Atoms/atoms-settings.spec.ts

```diff
@@ -0,0 +1,136 @@
+import { Page, test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  TopPanelButton,
+  takeEditorScreenshot,
+  waitForPageInit,
+  pressButton,
+  drawBenzeneRing,
+  clickInTheMiddleOfTheScreen,
+  openFileAndAddToCanvas,
+  resetCurrentTool,
+  selectRing,
+  RingButton,
+  getAtomByIndex,
+} from '@utils';
+
+async function setHydrogenLabelsOn(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Atoms', { exact: true }).click();
+  const deltaX = 0;
+  const deltaY = 60;
+  const anyX = 638;
+  const anyY = 524;
+  await page.mouse.move(anyX, anyY);
+  await page.mouse.wheel(deltaX, deltaY);
+  await page.getByRole('button', { name: 'Terminal and Hetero' }).click();
+  await page.getByTestId('On-option').click();
+  await pressButton(page, 'Apply');
+}
+async function selectExtendedTableElements(page: Page, element: string) {
+  await page.getByTestId('extended-table').click();
+  await page.getByRole('button', { name: element, exact: true }).click();
+  await page.getByRole('button', { name: 'Add', exact: true }).click();
+}
+async function atomDefaultSettings(page: Page) {
+  await selectTopPanelButton(TopPanelButton.Settings, page);
+  await page.getByText('Atoms', { exact: true }).click();
+  const deltaX = 0;
+  const deltaY = 150;
+  const anyX = 638;
+  const anyY = 524;
+  await page.mouse.move(anyX, anyY);
+  await page.mouse.wheel(deltaX, deltaY);
+}
+
+async function ringBondCountQuery(page: Page, menuItem: string) {
+  await page.getByText(menuItem).click();
+  await page.locator('button:nth-child(5)').first().click();
+}
+
+async function substitutionCountQuery(page: Page, menuItem: string) {
+  await page.getByText(menuItem).click();
+  await page.getByRole('button', { name: 'As drawn' }).nth(1).click();
+}
+
+async function aromacitilityQuery(page: Page, menuItem: string) {
```

Comment:
there is no 'aromacitility' word, please change it

Reply:
Changed